docs/examples/compose/compose.yaml

```diff
@@ -0,0 +1,73 @@
+services:
+  postgresql:
+    image: postgres:16
+    healthcheck:
+      test: ["CMD", "pg_isready", "-q", "-U", "docs", "-d", "docs"]
+      interval: 1s
+      timeout: 2s
+      retries: 300
+    env_file:
+    - env.d/postgresql
+    - env.d/common
+    environment:
+    - PGDATA=/var/lib/postgresql/data/pgdata
+    volumes:
+    - ./data/databases/backend:/var/lib/postgresql/data/pgdata
+
+  redis:
+    image: redis:5
+
+  backend:
+    image: lasuite/impress-backend:latest
+    user: ${DOCKER_USER:-1000}
+    restart: always
+    environment:
+    - DJANGO_CONFIGURATION=Production
+    env_file:
+    - env.d/backend
+    - env.d/postgresql
+    - env.d/common
```

Comment:
I believe common needs to come before backend, otherwise you get errors like this
```
WARN[0000] The "DOCS_HOST" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCS_HOST" variable is not set. Defaulting to a blank string. 
WARN[0000] The "S3_HOST" variable is not set. Defaulting to a blank string. 
```